src/events.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "events_listener.h"
+#include "game_object.h" 
+#include "graphics_handler.h"
+#include "observable.h"
+#include <SDL2/SDL.h>
+
+class Events : public GameObject, public Observable<EventsListener> {
```

Comment:
If I'm understanding this correctly, this `Events` class is for each kind of event and isn't a single controller class, in that case non plural would be better?